packages/clerk-js/src/core/clerk.ts

```
@@ -2090,15 +2113,20 @@ export class Clerk implements ClerkInterface {
     }
   };
 
-  #loadInStandardBrowser = async (): Promise<boolean> => {
+  #loadInStandardBrowser = async (): Promise<void> => {
```

We might want to add JSDoc explaining this will now throw instead of always returning a boolean

---

`#loadInStandardBrowser` would throw even before this PR. It's not that this PR is changing this behaviour.

---

Ok, this makes more sense then 👍🏼